apps/framework-cli/src/cli.rs

```diff
@@ -104,6 +103,13 @@ async fn top_command_handler(
                 &settings
             );
 
+            if name.contains('.') {
```

Comment:
If this is about docker compose project name, I suggest removing special characters there.

Reply:
Good call - fixed it